cmd/clusterctl/README.md

```diff
@@ -50,6 +50,9 @@ If you are using minikube, to choose a specific minikube driver, please use the
 
 Additional advanced flags can be found via help.
 
+Also, some environment is supported:
+`CLUSTER_API_MACHINE_READY_TIMEOUT`: set this value to adjust the timeout value for machine to become ready, by default it's 30min, e..g set it like `export CLUSTER_API_MACHINE_READY_TIMEOUT=45` will make the timeout value become 45 min.
```

Comment:
```suggestion
`CLUSTER_API_MACHINE_READY_TIMEOUT`: set this value to adjust the timeout value in minutes for a machine to become ready. The default timeout is currently 30 minutes. `export CLUSTER_API_MACHINE_READY_TIMEOUT=45` will extend the timeout value to 45 minutes.
```

Reply:
@jichenjc can you please address this comment as well?

Reply:
oops forgot this one, thank for reminder..